docs/releasenotes/release-1.123.md

```diff
@@ -1,12 +1,6 @@
-# v1.123.0
+# v1.123.1
```

Comment:
If we're planning on not releasing 1.123.0 (or recommending 1.123.1 instead) a little note here saying "We do not recommend using 1.123.0, as it had an issue around ...".  Or if we never released it, saying "1.123.0 contained a regression that was caught during pre-release testing, and was not released.".  Otherwise people will wonder what happened to 1.123.0 !

Reply:
Thanks for the guide!